sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -236,6 +231,7 @@ export interface SubscribeOptions extends TracingOptions {
     maxBatchSize?: number;
     maxWaitTimeInSeconds?: number;
     ownerLevel?: number;
+    startPosition?: EventPosition | Map<string, EventPosition>;
```

Comment:
I like maps, but I wonder if we should allow a type like `{ [partitionId: string]: EventPosition }` as well? Passing in maps is a little awkward I think.

Reply:
A simpler type will indeed be easier to work with. Am ok with the above suggestion